tests/users/test_api_resend_email.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from flask import json
+
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1
+
+
+class TestResendEmailApi(BaseTestCase):
+    def setUp(self):
+        super(TestResendEmailApi, self).setUp()
+
+        self.verified_user = UserModel(
+            name=user1["name"] + "Example",
+            email=user1["email"],
+            username=user1["username"],
+            password=user1["password"],
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"],
+        )
```

Comment:
If I am not wrong, dictionary unpacking should work
```suggestion
        self.verified_user = UserModel(**user1)
```

Reply:
If that works fine, we will open an issue to refactor other places too.